content/applications/websites/website/configuration/google_search_console.rst

```diff
@@ -7,21 +7,155 @@ monitor, maintain, and troubleshoot their site's presence in Google Search resul
+===============
+
+.. note:: Google suggests creating at least one domain property to represent your site, as it is the
+   most comprehensive view of your website information. It will include all URLs in all subdomains
+   and across HTTP and HTTPS. It requires you to add the DNS record in your domain name provider,
+   but even if it takes some time, it is worth it.
+
+URL prefix property
+===================
+
+This type of verification is usually simpler as you have multiple verification methods, such as
+using your existing Google Analytics or Tag Manager account. It also makes sense to view a section
+of your website separately. For example, if you work with a consultant on a specific part of your
+website, you might want to verify this part separately to limit access to your data.
+
+.. note:: The best method for you depends on your comfort level and technical expertise. For
+   beginners, using a file upload or HTML tag might be easiest. Those options are convenient if you
+   already use Google Analytics or Tag Manager. You'll need to access your domain registrar's
+   settings for domain verification.
+
+
+Five ways to verify your site ownership
+=======================================
+
+.. _GSC-DNS-Record:
+
+DNS Record
+----------
+
+With this method, you add a DNS TXT record to your domain's DNS settings. Google provides you with a
+unique string to add as a TXT record. Once added, Google verifies ownership by checking the DNS
+settings.
+Start by copying the text record from the dialog google-site-verification=123aBc… after entering
+your domain.
+After you change your DNS record, go back to Search Console and click Verify in the verification
+pane. Once again, if the verification does not work immediately, wait a few hours or a day and try
+again, as the change may take some time to take effect. Just choose the unverified property from the
+:guilabel:`Properties` drop-down and it will try to verify you automatically.
+
+.. important:: To stay verified, do not remove the DNS Record even after verification succeeds.
+
+.. _GSC-HTML-File-Upload:
+
+HTML File Upload
+----------------
+
+This method involves uploading an HTML file provided by Google to the root directory of your
+website. Google verifies ownership by checking for this file. You will typically download the HTML
+verification file from Google Search Console, and then upload it to your website. Here, you just
+have to copy-paste the code in the appropriate field in the Website Settings. You will need to have
+permission to upload the file to your website where it can be accessed by a browser. Once you added
+your website URL under the URL prefix option and clicked on :guilabel:`continue`, expand the HTML
+file section where you will find a file :guilabel:`Download` button.
+
+.. image:: google_search_console/html-file-download.png
+   :alt: HTML file download
+
+Download your HTML verification file and upload it to the root directory of the website you want to
+verify.
```

Comment:
```suggestion
#. Download your HTML verification file and upload it to the root directory of the website you want to
verify.
#. Open the file and copy the verification code, e.g., `google123abc.html`.
```
- Do people generally know how to access the root directory of their website? (sorry, I am *not* an expert on this at all)
- I assume when you say on line 99 "paste the code", you mean the one we can find when we open the html file? I added it here otherwise people might get confused where they need to copy the code from. Feel free to change this if I'm wrong :slightly_smiling_face: